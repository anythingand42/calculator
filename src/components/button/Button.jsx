import styled from 'styled-components';

const StyledButton = styled.button`
  min-height: 28px;
`;

export default StyledButton;
