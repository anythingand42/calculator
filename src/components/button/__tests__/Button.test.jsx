import React from 'react';
import { render } from '@testing-library/react';
import 'jest-styled-components';

import Button from '../Button';

test('Button component', () => {
  const { container } = render(<Button />);
  expect(container.firstChild).toMatchSnapshot();
});
