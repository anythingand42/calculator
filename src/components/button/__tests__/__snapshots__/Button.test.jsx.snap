// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component 1`] = `
.c0 {
  min-height: 28px;
}

<button
  class="c0"
/>
`;
